benchmark/CMakeLists.txt

```diff
@@ -45,8 +45,7 @@ function(add_umf_benchmark)
         ${BENCH_NAME}
         PRIVATE ${UMF_CMAKE_SOURCE_DIR}/include
                 ${UMF_CMAKE_SOURCE_DIR}/src/utils
-                ${UMF_CMAKE_SOURCE_DIR}/test/common
-                ${UMF_CMAKE_SOURCE_DIR}/examples/common)
+                ${UMF_CMAKE_SOURCE_DIR}/test/common ${LEVEL_ZERO_INCLUDE_DIRS})
```

Comment:
add L0 include dirs below when we check if(UMF_BUILD_LEVEL_ZERO_PROVIDER)

Reply:
Done